File: usar_banco.py
# coding: utf8
import os
import sys
import ConfigParser

ini_file = '/Ello/Windows/Ello.ini'
db_file = sys.argv[1].replace('\\', '/')

config = ConfigParser.RawConfigParser()
config.read(ini_file)
config.set('Dados', 'DataBase', db_file)

with open(ini_file, 'w') as config_file:
    config.write(config_file)

print 'Arquivo {0} configurado para usar o banco {1}'.format(ini_file, db_file)

File: usar_banco.c
#include <stdio.h>
#include <windows.h>

int main(int argc, char* argv[])
{
    if(argc < 2) {
        puts("Configura o arquivo .ini do Ello para utilizar o banco de dados informado.");
        puts("Uso: usar_banco <nome do banco de dados>");
        return 1;
    }
    WritePrivateProfileString("Dados", "Database", argv[1], "C:\\ello\\windows\\ello.ini");
    return 0;
}